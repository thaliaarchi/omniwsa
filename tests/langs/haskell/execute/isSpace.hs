import Data.Char (isSpace, chr)
import Text.Printf (printf)

main :: IO ()
main = do
    let chars = [c | c <- [0..0x10FFFF], isSpace (chr c)]
    putStrLn "Characters matching Data.Char.isSpace:"
    mapM_ (\c -> printf "'%c' U+%04X\n" (chr c) c) chars
