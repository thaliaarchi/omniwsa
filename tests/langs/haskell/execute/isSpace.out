The Glorious Glasgow Haskell Compilation System, version 9.12.1
Characters matching Data.Char.isSpace:
'	' U+0009
'
' U+000A
'' U+000B
'' U+000C
'' U+000D
' ' U+0020
' ' U+00A0
' ' U+1680
' ' U+2000
' ' U+2001
' ' U+2002
' ' U+2003
' ' U+2004
' ' U+2005
' ' U+2006
' ' U+2007
' ' U+2008
' ' U+2009
' ' U+200A
' ' U+202F
' ' U+205F
'　' U+3000
